### PR TITLE
Remove references to rhods, change to odh

### PR DESCRIPTION
**Pull request by jeff-phillips-18**

### CODE REVIEW

**Review by VaishnaviHire — COMMENTED** [review]
Do we need to update the references in [quickstart.ts](https://github.com/red-hat-data-services/odh-dashboard/blob/master/backend/src/routes/api/quickstarts/quickStartUtils.ts#L10) file as well ?

File: backend/src/routes/api/components/list.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { FastifyRequest } from 'fastify';
-import { KubeFastifyInstance, ODHApp } from '../../../types';
+import { KubeFastifyInstance, OdhApplication } from '../../../types';
```
[the remark this answers is not in the document]
Comment (jeff-phillips-18) [reply backend/src/routes/api/components/list.ts:2]: Trying to be more standardized. Going to be more important later when we use CRDs for these.

File: backend/src/routes/api/getting-started/gettingStartedUtils.ts [hunk 2]
```diff
@@ -22,9 +19,9 @@ export const getGettingStartedDoc = (appName: string): { appName: string; markdo
   }
 };
 
-export const getGettingStartedDocs = (): ODHGettingStarted[] => {
+export const getGettingStartedDocs = (): OdhGettingStarted[] => {
```
[the remark this answers is not in the document]
Comment (jeff-phillips-18) [reply backend/src/routes/api/getting-started/gettingStartedUtils.ts:22]: Same as above 😉

**Comment by jeff-phillips-18** [comment]
> Do we need to update the references in [quickstart.ts](https://github.com/red-hat-data-services/odh-dashboard/blob/master/backend/src/routes/api/quickstarts/quickStartUtils.ts#L10) file as well ?

That name is created by odh-manifests.

**Pull request merged**